lib/hacluster.pm

```diff
@@ -310,8 +311,11 @@ sub get_lun {
 
 sub pre_run_hook {
     my ($self) = @_;
-
-    $prev_console = $autotest::selected_console;
+    if (isotovideo::get_version() == 12) {
```

Comment:
that should probably be ">= 12".

Can you use a conditional assignment here?

```perl
$prev_console = isotovideo::get_version() >= 12 ? $autotest::selected_console : $testapi::selected_console;
```

or maybe this can both not work because perl tries to compile the code anyway?

Reply:
@alvarocarvajald [pointed out the same](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5416#discussion_r203326584)  :P, but in any case, seems to be one of the scripts in the makefile is complaining about (and I'm failing to reproduce that one locally on a tw machine)

Reply:
@foursixnine make sure that your working copy of os-autoinst that is referenced within os-autoinst-distri-opensuse is up-to-date. https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/Makefile#L12 clones os-autoinst … unless it's already checked out

Reply:
@okurz yeah... I just had the wrong version of parallel (had to install gnu_parallel instead of the one from moreutils...)

Reply:
@okurz for the time being I guess we'll have to live with this until https://github.com/os-autoinst/os-autoinst/pull/992 is merged and deployed. But this will enable @ldevulder and @alvarocarvajald to carry on with the testing